src/User/Controller/SecurityController.php

```diff
@@ -116,17 +116,26 @@ public function actionLogin()
             return $this->goHome();
         }
 
-        /** @var LoginForm $form */
+        /**
+* 
+         *
+ * @var LoginForm $form 
+*/
```

Comment:
Please review your IDE linting, these kinds of changes are badly formatted. The original format is preferred. In case of multiline comments, they should be properly indented